test/runtests.jl

```diff
@@ -0,0 +1,54 @@
+using Test
+
+function is_test_folder(f, path_separator)
+  a = split(f, path_separator)
+  dir = join(a[1:end-1], path_separator)
```

Comment:
You may want to look at `joinpath` it takes care of the path separator for you.

Reply:
Ah, ok, I'll update.